hls4ml/utils/config.py

```diff
@@ -144,7 +144,7 @@ def make_layer_config(layer):
             layer_config['Precision'] = default_precision
             layer_config['ReuseFactor'] = default_reuse_factor
             layer_config['table_size'] = 1024
-            if layer['config']['activation'] == 'softmax':
+            if layer_config['activation'] == 'softmax':
```

Comment:
With the latest changes in JSON format, `Softmax` and `ReLU` are layers themselves (i.e., they can appear outside of other layers not just as `Activation` layers). They should be added to `activation_layers`

Reply:
I've added that in the latest commit.